dqc_us_rules/dqc_us_0013.py

```diff
@@ -0,0 +1,135 @@
+    blacklist_facts = filter_negative_number_with_dependence_facts(val, blacklist_dict.keys())
+    for fact in blacklist_facts:
+        index_key = blacklist_dict[fact.qname.localName]
+        val.modelXbrl.error(
+            '{base_key}.{extension_key}'.format(
+                base_key=_CODE_NAME, extension_key=index_key
+            ),
+            messages.get_message(_CODE_NAME, str(index_key)),
+            concept=fact.concept.label(), modelObject=fact,
+            ruleVersion=_RULE_VERSION
+        )
+
+
+def filter_negative_number_with_dependence_facts(val, blacklist_concepts):
+    """
+    Checks the numeric, negative value facts in the provided ModelXBRL instance
+    against the rule dictionary and returns those which meet the conditions of the blacklist.
+
+    :param val: val whose modelXbrl provides the facts to check
+    :type val: :class:'~arelle.ModelXbrl.ModelXbrl'
+    :param blacklist_concepts: An iterable of the blacklist concepts we should
+        be testing against.
+    :type blacklist_concepts: list [str]
+    :return: Return list of the facts falling into the blacklist.
+    :rtype: list [:class:'~arelle.ModelInstanceObject.ModelFact']
+    """
+    blacklist_exclusion_rules = neg_num.get_rules_from_csv(_DEFAULT_EXCLUSIONS_FILE)
+    bad_blacklist = []
+    # Checks if the precondition concept exists and only proceeds with check if true
+    if dqc_13_precondition_check(val):
+        numeric_facts = facts.grab_numeric_facts(list(val.modelXbrl.facts))
+        # other filters before running negative numbers check
+        # numeric_facts has already checked if fact.value can be made into a number
+        facts_to_check = [
+            f for f in numeric_facts if decimal.Decimal(f.value) < 0 and
+            f.concept.type is not None and
+            # facts with numerical values less than 0 and contexts and
+            f.context is not None and
+            # check xsd type of the concept
+            f.isNumeric
+        ]
+
+        # identify facts which should be reported as included in the list
+        for fact in facts_to_check:
+            if neg_num.check_rules(fact, blacklist_exclusion_rules):
+                continue  # cannot be black
+            if fact.qname.localName in blacklist_concepts:
+                bad_blacklist.append(fact)
+
+    return bad_blacklist
+
+def dqc_13_precondition_check(val):
+    """
+    Checks if the precondition fact(s) exist and grabs their values.  Runs through an
+    if/else statement of the precondition check and totals the values to ensure they
+    are greater than zero.
+
+    :param val: val whose modelXbrl provides the facts to check
+    :type val: :class:'~arelle.ModelXbrl.ModelXbrl'
+    :return: True or False depending on the precondition check
+    :rtype: bool
+
+    """
+    facts_list = list(val.modelXbrl.facts)
+    total = 0
+
+    check1, value1 = facts.precondition_fact_exists(facts_list, _PRECONDITION_ELEMENT_1)
```

Comment:
It's quite likely that I am completely overlooking how to alter the code to what you have described...but based on the if/else statement that determines the precondition value (I have included the statement below), I am not coming up with a way to sum the values using the for loop/if statement suggested?

if (exists(IncomeLossFromContinuingOperationsBeforeIncomeTaxesExtraordinaryItemsNoncontrollingInterest.value))
                                        IncomeLossFromContinuingOperationsBeforeIncomeTaxesExtraordinaryItemsNoncontrollingInterest.value
                                else                                     if(exists(IncomeLossFromContinuingOperationsBeforeIncomeTaxesMinorityInterestAndIncomeLossFromEquityMethodInvestments.value))
                                        IncomeLossFromContinuingOperationsBeforeIncomeTaxesMinorityInterestAndIncomeLossFromEquityMethodInvestments.value + 
IncomeLossFromEquityMethodInvestments.value
                                        else
                                                if        (exists(IncomeLossFromContinuingOperationsBeforeIncomeTaxesDomestic.value) or                                                     exists(IncomeLossFromContinuingOperationsBeforeIncomeTaxesForeign.value))
                                                        IncomeLossFromContinuingOperationsBeforeIncomeTaxesDomestic.value +                                                 IncomeLossFromContinuingOperationsBeforeIncomeTaxesForeign.value +
IncomeLossFromEquityMethodInvestments.value
                                                else
                                                       0)

NOTE:  The precondition value if-else statement can be found here:  https://docs.google.com/spreadsheets/d/12aK2F3twvraeU7plHn0H7DjshmEYI5dKGuN8uB6efLQ/edit?ts=5720d837#gid=1473390933